presto-main/src/test/java/com/facebook/presto/sql/planner/iterative/rule/TestMergeAdjacentWindows.java

```diff
@@ -41,10 +43,24 @@
 import static com.facebook.presto.sql.planner.assertions.PlanMatchPattern.window;
 import static com.facebook.presto.sql.tree.FrameBound.Type.CURRENT_ROW;
 import static com.facebook.presto.sql.tree.FrameBound.Type.UNBOUNDED_PRECEDING;
+import static io.airlift.testing.Closeables.closeAllRuntimeException;
 
 public class TestMergeAdjacentWindows
 {
-    private final RuleTester tester = new RuleTester();
```

Comment:
Searching for `private final RuleTester tester = new RuleTester();` yields 6 such places in `master`. Is this one special?